modules/SofaMiscFem/FastTetrahedralCorotationalForceField.inl

```diff
@@ -155,6 +155,11 @@ template <class DataTypes> FastTetrahedralCorotationalForceField<DataTypes>::Fas
     , f_youngModulus(initData(&f_youngModulus,(Real)1000.,"youngModulus","Young modulus in Hooke's law"))
     , lambda(0)
     , mu(0)
+    , f_drawing(initData(&f_drawing, true, "drawing", " draw the forcefield if true"))
```

Comment:
This rise some question to me about the general consistency of drawinging in Sofa...but this is far 
outside the scope of this PR.

Reply:
I took the code that is in TetrahedralCorotationalForcefield to be consistent. Is it a good way to do it, this is another discussion...